sdk/redisenterprise/Azure.ResourceManager.RedisEnterprise/CHANGELOG.md

```diff
@@ -3,8 +3,11 @@
 ## 1.3.0-beta.1 (Unreleased)
```

Comment:
Please also update the `Unreleased` here with your desired release date.
In the meantime, since you are updating the library to a stable api-version, if you would like a stable version release, please update the version here **and** the version in `csproj` file to `1.3.0`.